presto-spi/src/main/java/com/facebook/presto/spi/block/Block.java

```diff
@@ -305,4 +305,9 @@ default Block getLoadedBlock()
     {
         return this;
     }
+
+    /**
+     * Returns a block that has an appended null at the end. The original block won't be modified.
```

Comment:
@yingsu00 Would you clarify what happens if original block already has a null?

Reply:
Modified it to
```
   /**
     * Returns a block that has an appended null at the end, no matter if the original block has null or not. 
     * The original block won't be modified.
     */
```